app/src/main/kotlin/com/exallium/todoapp/mvp/BasePresenter.kt

```diff
@@ -8,7 +8,7 @@ import rx.subscriptions.CompositeSubscription
  * The Presenter is considered "alive" when it is created, but "dead" as soon as
  * the underlying view is destroyed.
  */
-abstract class BasePresenter<out V : BaseView>(view: V) {
+abstract class BasePresenter<out V : BaseView>(private val view: V) {
```

Comment:
@jshvarts yes, make the view param protected.  no need to set it more than once I think